src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php

```diff
@@ -86,6 +88,19 @@ protected static function bootKernel(array $options = [])
         return static::$kernel;
     }
 
+    /**
+     * This is the best way to get a dependency injection container. It has access
```

Comment:
`Provides a dedicated test container with access to public and private services.`?

Reply:
Thank you